pytorch_lightning/trainer/data_loading.py

```diff
@@ -109,6 +109,7 @@ def _worker_check(self, dataloader: DataLoader, name: str) -> None:
 
     def auto_add_worker_init_fn(self, dataloader: DataLoader) -> None:
         if int(os.environ.get("PL_SEED_WORKERS", 0)) and dataloader.worker_init_fn is None:
+            rank_zero_info("Setting dataloader init function to seed all workers")
```

Comment:
What's the benefit of this message?

Reply:
> What's the benefit of this message?

You can see the discussion in the [GitHub issue](https://github.com/PyTorchLightning/pytorch-lightning/issues/7937). It's impossible to know whether the workers have been seeded correctly or not, so it would be great to have some log message that confirms that the seed has been set. The best way to confirm this is to log a message from every worker of every process. We don't want to flood the log with so many info-level messages, so that's why we write debug-level messages in `pl_worker_init_function()` and just this one info-level message here. Maybe this info-level message is redundant? I think it's most important that I can turn on debug messages to check that the it's working.

Reply:
> Maybe this info-level message is redundant?

I think it is and should be `debug` level. But then you already have the one inside `pl_worker_init_function` so no point in having both.

Reply:
Removed.